src/test/java/com/xiaomi/infra/pegasus/rpc/async/MetaSessionTest.java

```diff
@@ -306,7 +307,7 @@ public void testDNSResetMetaMaxQueryCount() {
   public void testDNSMetaUnavailable() {
     // Ensures when the DNS returns meta all unavailable, finally the query will timeout.
     ClusterManager manager =
-        new ClusterManager(1000, 4, false, null, 60, new String[] {"localhost:34601"});
+        new ClusterManager(1000, 4, false, null, 60, new String[] {"localhost:34601"}, false);
```

Comment:
Too many params in the constructor of `ClusterManager`, we can refactor this class later using [Builder Pattern](https://en.wikipedia.org/wiki/Builder_pattern).

Reply:
Sometime, Builder Pattern isn't also a good choice which is tedious, especially new config need add. I think the viewpoint of first answer: “[builder](https://www.zhihu.com/question/326142180/answer/697172067)” may be referenced?